### ISSUE TITLE
Unable to decrypt configuration items

### ISSUE THREAD

**Issue by ulsa** [awslabs/aws-lambda-redshift-loader#16]
I get an error after the manifest has been written, during decryption:

```
Error during Decryption
Unable to decrypt configuration items due to
{ [InvalidCiphertextException: null]
  message: null, 
  code: 'InvalidCiphertextException', 
  time: Wed Apr 15 2015 12:40:32 GMT+0000 (UTC), 
  statusCode: 400, 
  retryable: false, 
  retryDelay: 30 }
Error Message: InvalidCiphertextException: null
error:
{ "errorMessage": "error" }
```

The configuration was created using the `setup.js` script, although some fields, eg `s3Prefix`, have been manually edited in DynamoDB.

**Comment by IanMeyers** [comment]
Can you check if you had set the AWS_REGION environment variable to match that of your Lambda function when you ran setup.js?

**Comment by ulsa** [comment]
The lambda is in eu-west-1, and I exported AWS_REGION to that before I ran setup. The key is located in eu-west-1. There is no key in any other region.

I did run setup once with AWS_REGION=... on the command line, but not exported before. That time it did hang on encrypting the password (see issue #12). Not sure what that could have caused though.

**Comment by IanMeyers** [comment]
Sorry to ask it of you, but can you please delete the configuration and re-enter it. I'm going to add a resetPassword utility now...

**Comment by ulsa** [comment]
I deleted the config item from dynamodb, then reran the setup script. Still the same problem on decryption. However, the key seems to be the same. Perhaps I should have deleted that too?

**Comment by yang-zhang** [comment]
I have the same issue, also on eu-west-1.

**Comment by IanMeyers** [comment]
Hello,

We've just released version 2.0.0 which includes an 'encryptValue.js' which should allow you to verify that AWS Key Management Service connectivity is working OK, and producing the correct values. Can you please download this and try encrypting your password to validate that you have the correct rights and that you are getting the expected output?

Thx,

Ian
